test/api/index.ts

```diff
@@ -33,7 +33,12 @@ describe('RippleAPI [Test Runner]', function() {
   const filteredTestSuites = allTestSuites.filter(({ isMissing }) => !isMissing)
 
   // Run all the tests:
-  for (const { name: suiteName, tests, isMissing } of filteredTestSuites) {
+  for (const {
+    name: suiteName,
+    tests,
+    isMissing,
+    config
```

Comment:
we added support for an `export config = {}` in the test files, to tell the test runner how they should be run. We needed this because one of the test files tested the mocked server in a way that always returned a non-X style address, even if you passed an X-style address. Skipped these for now while we finish up this migration, and then can revisit how we handle mocked responses.